Product/Production/Common/CONNECTCoreLib/src/test/java/gov/hhs/fha/nhinc/callback/opensaml/HOKSAMLAssertionBuilderTest.java

```diff
@@ -369,10 +370,7 @@ public void testCreateAuthenticationDecisionStatements() throws PropertyAccessEx
         assertEquals(issuer.getFormat(), SAMLAssertionBuilder.X509_NAME_ID);
 
         final Conditions conditions = assertion.getConditions();
-        assertTrue(
-            beforeCreation.isBefore(conditions.getNotBefore()) || beforeCreation.isEqual(conditions.getNotBefore()));
-        assertTrue(beforeCreation.isBefore(conditions.getNotOnOrAfter())
-            || beforeCreation.isEqual(conditions.getNotOnOrAfter()));
+        assertNull(conditions);
```

Comment:
We are just asserting on null here. Could we have some more meaningful values to test for?